static/scss/auth.scss

```diff
@@ -160,6 +196,34 @@
   .col {
     align-self: flex-start;
   }
+
+  .form-control:focus {
+    border-color: $success;
+    box-shadow: 0 0 0 1px $success;
+  }
+
+  label {
+    font-weight: normal !important;
+    justify-content: space-between;
+
+    div {
+      font-weight: inherit !important;
+    }
+
+    .subtitle {
+      color: $label-subtitle;
+      font-size: 14px;
+    }
+
+    .red {
```

Comment:
Can you change this class name from `red` to `required`? What we're really styling here is a character that indicates the field is required, so that's a more useful class name, and it won't be a problem if the actual color changes at some point